docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
-[//]: # (title: Output formats supported by Kotlin Notebooks)
+[//]: # (title: Output formats supported by Kotlin Notebook)
 
-There will be examples for each output.
+Kotlin Notebook supports a variety of output types, including text, HTML, and images. With the help of external libraries,
```

Comment:
Please mention somewhere that the user can use `DISPLAY` function to display any kind of the output. For example, here I display multiple plots in one output:
<img width="702" alt="image" src="https://github.com/JetBrains/kotlin-web-site/assets/20319593/f86579ba-d4bd-407f-8280-61915d918a8d">

In 2024.1 it will be also possible to update outputs


Uploading Screen Recording 2024-03-06 at 01.03.25.mov…